src/Http/Routing/WebRouting.php

```diff
@@ -62,5 +63,9 @@ public function wire(): void
             $this->router->put("/speakers/{speaker}", "update")->name("speakers.update");
             $this->router->delete("/speakers/{speaker}", "destroy")->name("speakers.destroy");
         });
+
+        $this->router->controller(AssetsController::class)->group(function (): void {
+            $this->router->get("/static/{folder}/{file}", "index")->name("assets");
```

Comment:
What about more complicated path structures?

Reply:
This should be helpful: https://laravel.com/docs/8.x/routing#parameters-encoded-forward-slashes